js/src/rive_advanced.mjs.d.ts

```diff
@@ -1,10 +1,15 @@
 interface RiveOptions {
-    locateFile(file: string): string
+    locateFile(file: string): string;
   }
-  
+
   declare function Rive(options?: RiveOptions): Promise<RiveCanvas>;
   export default Rive;
-  
+
+  /**
+   * RiveCanvas is the main export object that contains references to different Rive classes to help
+   * build the Rive render loop for low-level API usage. In addition, this contains multiple methods that help
+   * aid in setup, such as loading in a Rive file, creating the renderer, and starting/finishing the render loop (requestAnimationFrame)
```

Comment:
Line length? What column limit are we using here?

Reply:
Prettier defaults to 80, I totally missed this didn't apply to comments on save.

But on further look, seems like they [don't enforce](https://prettier.io/docs/en/rationale.html#print-width) on comments. So we can configure eslint to error out if past like 100.